src/modules/esp32/PaxcounterModule.cpp

```diff
@@ -7,21 +7,39 @@
 
 PaxcounterModule *paxcounterModule;
 
-void NullFunc(){};
-
-// paxcounterModule->sendInfo(NODENUM_BROADCAST);
+/**
+ * Callback function for libpax.
+ * We only clear our sent flag here, since this function is called from another thread, so we
+ * cannot send to the mesh directly.
+ */
+void PaxcounterModule::handlePaxCounterReportRequest()
+{
+    // The libpax library already updated our data structure, just before invoking this callback.
+    LOG_INFO("PaxcounterModule: libpax reported new data: wifi=%d; ble=%d; uptime=%lu\n",
+             paxcounterModule->count_from_libpax.wifi_count, paxcounterModule->count_from_libpax.ble_count, millis() / 1000);
+    paxcounterModule->reportedDataSent = false;
+}
```

Comment:
you can trigger the osthread processing here by calling paxcounterModule->setIntervalFromNow(0);